### PR TITLE
Release 87 patches

### PR DESCRIPTION
**Pull request by premanand17**

### CODE REVIEW

File: sql/patch_86_87_b.sql [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+# patch_86_87_a.sql
+#
+# Title: Meta case insensitive
+#
+# Description:
+# Enfored meta_value in meta table to be NOT NULL
```
Comment (avullo) [inline sql/patch_86_87_b.sql]: Being pedantic here: "Enforce"

**Comment by nerdstrike** [comment]
That all looks pretty reasonable to me.

File: sql/patch_86_87_b.sql [hunk 2]
```diff
@@ -13,9 +13,9 @@
...
 -- limitations under the License.
 
-# patch_86_87_a.sql
+# patch_86_87_b.sql
 #
-# Title: Meta case insensitive
+# Title: Meta value NOT NULL
 #
 # Description:
 # Enfored meta_value in meta table to be NOT NULL
```
Comment (avullo) [inline sql/patch_86_87_b.sql]: Enforce

**Comment by lairdm** [comment]
Good first steps, relco padawan.

**Pull request merged**